public/index.html

```diff
@@ -134,9 +134,10 @@
                     this site in your browser using the shortcut below:
                 </p>
                 <pre class="static-pre">
-                MacOS: Cmd + R
-                Linux: Ctrl + Shift + R
-                Windows: Ctrl + F5
+                macOS : option + command + R (Safari)
+                macOS : shift + command + R (Chrome, Firefox)
+                Linux : shift + control + R
+                Windows : Ctrl + Fn + F5
```

Comment:
Shouldn't this be control as well, instead of Ctrl?

Also, the function modifier is just for laptops in order to actually use function keys, so it's implied, and also implementation-dependent (sometimes Fn modifier is used to disable the function keys)

Reply:
@veggiesaurus for windows, it is changed to control + F5. I actually don't have a windows system here... could you confirm "control+F5" is _hard_ reload instead of the regular reload?